### DISCUSSION TITLE
antd@5样式被覆盖

### DISCUSSION THREAD

**Discussion by WT-Road** [umijs/umi#10271] [Q&A]
Antd@5样式正常, border-radius: 6px;
![image](https://user-images.githubusercontent.com/14371925/211735221-583ebdc1-6b20-48df-a5df-1d605cd5bce9.png)

使用antd@5的token设置borderRadius: 18,样式被mf-dep____vendorxxxx.css文件覆盖
![image](https://user-images.githubusercontent.com/14371925/211735377-be316f6e-84e5-44a9-a2a6-60a1155d7398.png)

疑问这个mf-dep____vendor这个文件是哪来的,怎么关闭呢
"umi": "^4.0.34",
"antd": "5.1.4",

**Comment by fz6m** [comment]
没遇到你说的问题，给一个最小复现看看。

**Reply by WT-Road** [reply]
> 没有遇到你说的问题，给一个最小的复现看。

谢谢,找到原因了,我移除了"@ant-design/pro-layout": "^6.38.22",样式正常了
